Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -38,37 +61,36 @@ def binary_setcap(self, binaryPath):
         if os.path.exists(AGENT_SCRIPT_PATH) and os.system('sudo %s %s' %(AGENT_SCRIPT_PATH, binaryPath)) == 0:
             return True
         else:
-            LG().Log('ERROR', 'Error setting capabilities to npmd agent binary.')
+            LG().Log(LogType.Error, 'Error setting capabilities to npmd agent binary.')
```

Comment:
Yes some places we are not intent on sending the log. Like this one because a verification of agent capabilities is done at the plugin end before starting the agent.